harness/determined/profiler.py

```diff
@@ -100,17 +100,20 @@ def __init__(
 
         # If the ProfilingAgent is disabled, don't waste resources by creating useless threads
         if self.is_enabled:
-            # Set up timer thread to stop collecting after 5 minutes
-            self.max_collection_seconds = 300
-            self.shutdown_timer = PreemptibleTimer(
-                self.max_collection_seconds, self._end_collection
-            )
-
-            # Set up the thread responsible for making API calls
+            # Set up sender thread with the number of producers to expect shutdowns from.
             self.send_queue = (
                 queue.Queue()
             )  # type: """queue.Queue[Union[List[TrialProfilerMetricsBatch], ShutdownMessage]]"""
-            self.sender_thread = ProfilerSenderThread(self.send_queue, self.master_url)
+
+            # Set up timer thread to stop collecting after 5 minutes
+            self.shutdown_timer = PreemptibleTimer(MAX_COLLECTION_SECONDS, self._end_collection)
```

Comment:
nit: is there a reason creating the `shutdown_timer` was moved down to be between `self.send_queue` and creating `self.sender_thread`?

Reply:
I will fix. I literally just was moving where send queue was made around so that it would be within line length since i couldn't figure out how to write a multiline `# type` comment